client.go

```diff
@@ -272,3 +295,70 @@ func (cl *Client) Namespaces(ctx context.Context) ([]Namespace, error) {
 
 	return ns, nil
 }
+
+// Users get list of users by id.
+func (cl *Client) Users(ctx context.Context, ids ...int) (map[int]User, error) {
+	ususerids := []string{}
+
+	for _, id := range ids {
+		ususerids = append(ususerids, strconv.Itoa(id))
+	}
+
+	body := url.Values{
+		"action":        []string{"query"},
+		"list":          []string{"users"},
+		"usprop":        []string{"groups|editcount|groupmemberships|registration|emailable"},
+		"format":        []string{"json"},
+		"formatversion": []string{"2"},
+		"ususerids":     []string{strings.Join(ususerids, "|")},
+	}
+
+	data, status, err := req(
+		ctx,
+		cl.httpClient,
+		http.MethodPost,
+		fmt.Sprintf("%s%s", cl.url, cl.options.UserURL),
+		strings.NewReader(body.Encode()),
+		map[string]string{
+			"Content-Type": "application/x-www-form-urlencoded",
+		})
+
+	if err != nil {
+		return nil, err
+	}
+
+	if status != http.StatusOK {
+		return nil, fmt.Errorf(errBadRequestMsg, status, data)
+	}
+
+	res := new(userResponse)
+
+	if err := json.Unmarshal(data, res); err != nil {
+		return nil, err
+	}
+
+	users := make(map[int]User)
+
+	for _, user := range res.Query.Users {
+		if !user.Missing {
+			users[user.UserID] = user
+		}
+	}
+
+	return users, nil
+}
+
+// Users get single user by id.
+func (cl *Client) User(ctx context.Context, id int) (User, error) {
```

Comment:
Do we need an extra API for this? Can't we just call Users with ids slice containing one element?

Reply:
This is done in order to save user of the package from `boilerplate code`. If you decide to call `Users` functions to get users, you need:
a) Check for the error that is returned from the method 
b) Check if `map` that was returned from the method contains that particular id
c) If you want to notify parent function that user was not found you need to create custom error type for that etc.
This makes code that uses `Users`  for single user API call cumbersome and repeatable especially if you want to call in couple handlers. So intention of this piece of code is to solve that problem beforehand by providing convenience function to get single user.

Reply:
Makes sense